### PR TITLE
[DES-326] (SCSS Files for dashboard) Change Gotham font to Metropolis font

### PR DESCRIPTION
**Pull request by levadadenys**
### [DES-326] (SCSS Files for dashboard) Change Gotham font to Metropolis font

* change Gotham font to Metropolis Font across dashboard/apps directories in scss
* added font-weights variables
* added font-weights variables

Will be a part of bigger PR https://github.com/code-dot-org/code-dot-org/pull/53756

## Links
- jira ticket: [DES-326](https://codedotorg.atlassian.net/browse/DES-326)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/style/common.scss [hunk 1]
```diff
@@ -1655,11 +1646,11 @@ $see-examples-thick-underline-color: $yellow;
 
 .droplet-block-tooltipster .tooltipster-content {
   font-family: $tooltip-font;
+  font-weight: 400;
```
Comment (sanchitmalhotra126) [inline apps/style/common.scss]: Should we store separate constants for font weight as well?

File: apps/style/craft/style.scss [hunk 2]
```diff
@@ -546,11 +546,12 @@ $height-to-width: math.div(434, 477);
     @include pixelated;
     @include gray-button;
     @include mc-text-shadow;
+    @include main-font-semi-bold;
 
+    font-family: $main-font !important;
```
Comment (sanchitmalhotra126) [inline apps/style/craft/style.scss:551]: Why do we need both of these? I'm guessing the !important would override right?

**Review by sanchitmalhotra126 — APPROVED** [review]
Couple minor questions but looks good overall!

File: apps/style/craft/style.scss (hunk 2 above)
Comment (levadadenys) [reply apps/style/craft/style.scss:551]: `!important` is here to create the same priority scss rule (before the change of fontFamily `!important` was already here)

I've also used mixin cause right now we also need to set font weith and font styles alongside with font family

File: apps/style/common.scss (hunk 1 above)
Comment (levadadenys) [reply apps/style/common.scss]: Not sure if we'll be using it often, but yeah, I guess that a good idea, will do, thank you!

Comment (levadadenys) [reply apps/style/common.scss]: Added

**Pull request merged**